### ISSUE TITLE
[DOCS] <Missing  links in pages/How.md file>

### ISSUE THREAD

**Issue by 1Harshini** [Pradumnasaraf/open-source-with-pradumna#9]
### Description

Missing  links in [How.md file](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How.md)

**Comment by Pradumnasaraf** [comment]
Hey Harshini, I didn't get your missing link problem, are you talking about the empty file with no content?

**Comment by 1Harshini** [comment]
exactly, thought to add some resources in [how.md file](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How.md) directing some blogs and youtube videos

**Comment by Pradumnasaraf** [comment]
> exactly, thought to add some resources in [how.md file](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How.md) directing some blogs and youtube videos

Thank you for the consideration, I have added a page called resources for that purpose. The plan for this page is to write detailed instructions (text  + video+ image) on things like "How to create a PR", "How to open an Issue", etc Each in its own separate file, **what's are your thoughts on that?**

**Comment by 1Harshini** [comment]
yeah This is more **sensable** 
- [ ] How to raise an Issue
- [ ] Creating a branch to work on an issue
- [ ] How to create a PR

**Comment by Pradumnasaraf** [comment]
Thank you Harshini, Currently, I am working and refactoring that only, It will live (the structure) within 1hr, let me know your feedback :)

**Comment by 1Harshini** [comment]
yupp sure!
will go based on the structure 👍

**Comment by Pradumnasaraf** [comment]
Hey @1Harshini , you check out the new structure, looking forward to your feedback

**Comment by 1Harshini** [comment]
yes @Pradumnasaraf ,this is more perceptible and clear

**Comment by Pradumnasaraf** [comment]
> yes @Pradumnasaraf ,this is more perceptible and clear   
> 

Thank you!!

**Comment by 1Harshini** [comment]
Hey 
Can I work on it  to add text, videos and blogs

**Comment by Pradumnasaraf** [comment]
> Hey Can I work on it to add text, videos and blogs

For sure, I have Opened some issues with all the details (will be more soon) or if you have any other issues you can open a new one.
Again thank you for your interest

**Comment by vikasganiga05** [comment]
Hi Pradumnasaraf, my suggestion is to add a badge like the below one as a placeholder :point_down: 

![Work In Progress](https://img.shields.io/badge/Markdown%20Guide-WIP-yellow)
![Work In Progress](https://img.shields.io/badge/Markdown%20Guide-WIP-yellow?style=flat-square)
![Work In Progress](https://img.shields.io/badge/Markdown%20Guide-WIP-yellow?style=for-the-badge)

**Comment by Pradumnasaraf** [comment]
Hey! @1Harshini if you have any suggestions, ideas, or feedback. feel free to drop them in the discussions, for now, I am closing this issue
